api/stream/design.go

```diff
@@ -257,30 +265,17 @@ var _ = Service("stream", func() {
 	Method("UpdateSourceSettings", func() {
 		Meta("openapi:summary", "Update source settings")
 		Description("Update source settings.")
-		Result(SettingsResult)
+		Result(Task)
 		Payload(SourceSettingsPatch)
 		HTTP(func() {
 			PATCH("/branches/{branchId}/sources/{sourceId}/settings")
 			Meta("openapi:tag:configuration")
-			Response(StatusOK)
+			Response(StatusAccepted)
 			SourceNotFoundError()
 			ForbiddenProtectedSettingError()
 		})
 	})
 
-	Method("RefreshSourceTokens", func() {
-		Meta("openapi:summary", "Refresh source tokens")
-		Description("Each sink uses its own token scoped to the target bucket, this endpoint refreshes all of those tokens.")
-		Result(Source)
-		Payload(GetSourceRequest)
```

Comment:
Unused, token can be refreshed by disable/enable API calls (not implemented yet).